src/main/scad/laszlo-molnar/bdsm/molnar.bdsm.scad

```diff
@@ -0,0 +1,119 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+
+pin_height = 1.5;
+pin_r = 1.6;
+pin_hole_inset = 0.2;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+
+*box();
+*pieces();
+
+module box () {
+    box_piece();
+    translate([-30, 50, 0])
+    rotate([0, 0, 180])
+    box_piece();
+    
+    translate([10, -10, 0])
+    rotate([0, 0, 45])
+    box_piece();
+}
+
+module pieces() {
+    burr_plate([
+        ["xx", "x."],
+        ["..x|xxx", "...|.x."],
+        ["xxx|..x", "x.."],
+        ["x.|xx", "x.", "x."],
+        [".xx|xx."],
+        [".x|xx", "..|.x", "..|.x"]
```

Comment:
Should the two pieces of height 3 be rotated to lie flat?

Reply:
Updated
![image](https://user-images.githubusercontent.com/1962079/75352076-1a954780-58e4-11ea-8349-3c1905adc223.png)